examples/MNIST_scikit_saliency.ipynb

```diff
@@ -0,0 +1,572 @@
+      "text/plain": [
+       "<Figure size 720x360 with 10 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "import time\n",
+    "from sklearn.model_selection import train_test_split\n",
+    "from sklearn.linear_model import LogisticRegression\n",
+    "\n",
+    "t0 = time.time()\n",
+    "\n",
+    "# Split data into test and train sets\n",
+    "train_samples = 20000\n",
+    "\n",
+    "X_train, X_test, y_train, y_test = train_test_split(\n",
+    "    X, y, train_size=train_samples, test_size=10000)\n",
```

Comment:
I just realized that this could use a fixes random seed to result in consistent results from run to run
```suggestion
    "X_train, X_test, y_train, y_test = train_test_split(\n",
    "    X, y, train_size=train_samples, test_size=10000, random_state=0)\n",
```